chef/cookbooks/crowbar/recipes/prepare-services-shutdown.rb

```diff
@@ -0,0 +1,34 @@
+# Cookbook Name:: crowbar
+# Recipe:: prepare-os-upgrade
```

Comment:
Oh! This file should not be there, I've moved both parts to `prepare-upgrade-scripts.rb`